src/components/Frame/components/ContextualSaveBar/ContextualSaveBar.tsx

```diff
@@ -23,7 +21,9 @@ export function ContextualSaveBar({
   contextControl,
 }: ContextualSaveBarProps) {
   const i18n = useI18n();
-  const {logo} = useTheme();
+  // TODO: This behavior will be re-enabled in a separate PR.
+  const logo = undefined as {[key: string]: any} | undefined;
```

Comment:
Drawing attention here to ensure this is capture in the following PR: https://github.com/Shopify/polaris-react/pull/4667